test/cypress/integration/test.acceptance.js

```diff
@@ -134,6 +134,56 @@ describe('Shepherd Acceptance Tests', () => {
           .should('exist')
           .and('be.visible');
       });
+
+      it('can be disabled', () => {
+        const tour = setupTour(Shepherd, {}, () => [
+          {
+            id: 'test-step',
+            buttons: [
+              {
+                classes: 'disabled-button',
+                text: 'disabled',
+                disabled: true
+              },
+              {
+                classes: 'enabled-button',
+                text: 'enabled'
+              }
+            ]
+          }
+        ]);
+        tour.start();
+        tour.show('test-step');
+        cy.get('.shepherd-button.disabled-button')
```

Comment:
Would we be able to move these tests to a unit test of the shepherd-button component? They are fairly simple, and unit tests will add to coverage.

Reply:
Yes, this makes sense, I have moved them :)